roles/bind-keycloak-apb/tasks/main.yml

```diff
@@ -82,26 +82,26 @@
 - set_fact: CLIENT_CONFIG="{{ installation_details.content | from_json }}"
 
 # Create keycloak client secret
-- name: Creating {{ CLIENT_NAME }} client secret
+- name: Creating {{ CLIENT_ID }} client secret
   k8s_v1_secret:
-    name: "{{ keycloak_secret_name }}-{{ CLIENT_NAME | lower }}-{{ CLIENT_TYPE }}"
+    name: "{{ keycloak_secret_name }}-{{ CLIENT_ID | lower }}-{{ CLIENT_TYPE }}"
     namespace: '{{ namespace }}'
     annotations:
       configType: "json"
     labels:
       mobile: enabled
       serviceName: keycloak
-      clientId: "{{ CLIENT_NAME }}"
+      clientId: "{{ CLIENT_ID }}"
     string_data:
-      client_name: "{{ CLIENT_NAME }}"
-      client_id: "{{ CLIENT_ID }}"
+      clientName: "{{ CLIENT_ID }}"
+      clientId: "{{ GENERATED_CLIENT_ID }}"
```

Comment:
That can be very confusing. I think we can change this to `id` and also add the actual `clientId` here as per keycloak terminology in their [client representation](https://www.keycloak.org/docs-api/2.5/rest-api/index.html#_clientrepresentation)?

Reply:
Yeah I see where you're coming from. `clientName` makes sense to me, so I personally wouldn't change it. `clientId` is a bit confusing because we're labelling it above with the same key but different value, so I think `id` would probably be better.

Reply:
I guess this is how we ran into the CLIENT_NAME/CLIENT_ID issues in the first place haha